### PR TITLE
[WFLY-19910] Add logicalVersion to WildFly manifests

### PR DESCRIPTION
**Pull request by spyrkob**
Issue: https://issues.redhat.com/browse/WFLY-19910

### CODE REVIEW

File: pom.xml
Comment (rhusar) [inline pom.xml:586]: Rebase went bad.

**Review by rhusar — CHANGES_REQUESTED** [review]

File: preview/channel/pom.xml
Comment (rhusar) [inline preview/channel/pom.xml:163]: We are calling this distribution 'WildFly Preview' - notice the capital P.

**Review by rhusar — CHANGES_REQUESTED** [review]

**Comment by bstansberry** [comment]
@spyrkob @jamezp I suspect there's a needed update to the WildFly Maven Plugin to handle the new manifest schema

**Comment by jamezp** [comment]
I just released 5.1.0.Alpah2 today. I hope that's enough.

I'd really like to find a way where we can release these types of things independently.

**Comment by spyrkob** [comment]
@bstansberry I think this should be ready now

**Comment by jfdenise** [comment]
I have logged https://issues.redhat.com/browse/WFLY-20028 to track the WildFly Galleon Plugins upgrade to 7.3.1.Final

File: pom.xml
Comment (mchoma) [inline pom.xml:337]: I was told by @fabiobrz and @jbliznak that we are waiting for this line to be merged to ublock our testing with wildfly snapshot.

Please can we work to merge this and unblock wildfly snapshot testing.

**Review by jamezp — APPROVED** [review]

**Pull request merged**
